### DISCUSSION TITLE
Is it possible to add an increaseViewportBy option to the VirtuosoGrid component?

### DISCUSSION THREAD

**Discussion by nabigraphics** [petyosi/react-virtuoso#807] [Q&A] [answered]
I am currently using the VirtuosoGrid and Virtuoso components and I am wondering why the increaseViewportBy option is not available in the VirtuosoGrid component. Is the internal logic different between the two components?

**Comment by petyosi** [comment, marked as answer]
No, totally possible and easy. Just a matter of exposing the stream to the props. happy to accept a PR.
